packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolAncestors.test.ts

```diff
@@ -0,0 +1,54 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+import { MempoolTx } from '../../../src/category/blockchain'
+import waitForExpect from 'wait-for-expect'
+
+describe('getMempoolAncestors', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+  it('should getMempoolAncestors and return JSON object if verbose is true', async () => {
+    await waitForExpect(async () => {
+      const txId = await client.wallet.sendToAddress('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU', 0.0003)
+      const mempoolEntry: MempoolTx = await client.blockchain.getMempoolAncestors(txId, true)
+      const entryKey = Object.keys(mempoolEntry)[0]
```

Comment:
You should expect the array of keys to have more than a size of 0, ie. `expect(Object.keys(mempoolEntry)).toBeGreaterThan(0)` so that we can avoid any funny business where for some reason the entryKey is undefined and has a value